sphinx_gallery/_static/gallery.css

```diff
@@ -55,7 +55,9 @@ div.sphx-glr-thumbContainer {
 
 .sphx-glr-script-out pre {
   overflow-y: auto;
-  max-height: 30em; }
+  max-height: 30em;
+  background-color: #FFFFFE;
```

Comment:
It looks nice on my screen with my eyes. I don't mind it to be very light.
![screenshot](https://cloud.githubusercontent.com/assets/208217/11422564/43f8ec78-943d-11e5-9450-5b418ea55ad2.png)